teuthology/task/install.py

```diff
@@ -702,28 +702,45 @@ def rh_install(ctx, config):
     :param ctx: the argparse.Namespace object
     :param config: the config dict
     """
+    yaml_path = None
+    # Look for rh specific packages in <suite_path>/rh/downstream.yaml
+    if 'suite_path' in ctx.config:
+        ds_yaml = os.path.join(
+            ctx.config['suite_path'],
+            'rh',
+            'downstream.yaml',
+        )
+        if os.path.exists(ds_yaml):
+            yaml_path = ds_yaml
+    # default to user home dir if one exists
+    default_yaml = os.path.expanduser('~/downstream.yaml')
```

Comment:
This also needs to be documented, as it could be quite surprising.

Also, the path to downstream.yaml should be logged just to help people discover potential conflicts.